src/datepicker/locale.js

```diff
@@ -45,6 +51,12 @@ const locale = {
   timeSelectStartLabel: 'Start time',
   timePickerAriaLabel: 'Select a time',
   timezonePickerAriaLabel: 'Select a timezone',
+  selectedStartDate: 'Selected start date.',
+  selectedEndDate: 'Selected end date.',
+  notAvailable: 'Not available.',
```

Comment:
do you mind making the property names more specific? For example, here I would call it `dateNotAvailable`